kntest/pkg/cluster/ops/create.go

```diff
@@ -40,6 +39,35 @@ const (
 	projectKey        = "E2E:Project"
 )
 
+// Create creates a GKE cluster and configures gcloud after successful GKE create request
+func Create(o *RequestWrapper) (*clm.GKECluster, error) {
+	gkeClient := clm.GKEClient{}
+	clusterOps := gkeClient.Setup(o.Request)
+	gkeOps := clusterOps.(*clm.GKECluster)
+	if err := gkeOps.Acquire(); err != nil || gkeOps.Cluster == nil {
+		return nil, fmt.Errorf("failed acquiring GKE cluster: '%v'", err)
+	}
+
+	// At this point we should have a cluster ready to run test. Need to save
+	// metadata so that following flow can understand the context of cluster, as
+	// well as for Prow usage later
+	writeMetaData(gkeOps.Cluster, gkeOps.Project)
+
+	// set up kube config points to cluster
+	// TODO(chaodaiG): this probably should also be part of clustermanager lib
```

Comment:
Or not?

Reply:
I think this comment can be removed, but cc @chaodaiG for opinions.

Reply:
It doesn't really matter at this point, kntest will play it out

Reply:
Removed the comment.